### DISCUSSION TITLE
Could not load file or assembly CefSharp.Wpf when dependent assembly has lower version referenced than the application loading the assembly

### DISCUSSION THREAD

**Discussion by SauravGuha** [cefsharp/CefSharp#4587] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

111.2.20.0

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.Net 4.8

### Implementation

WinForms

### Reproduction Steps

I have adapter design pattern implemented. My adapter is referencing 'CefSharp.Wpf, Version=105.3.330.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138' but the application that is using the adapter is downloading Version=111.20.2.0. We want to have a single version of cefsharp across the application(always the latest), irrespective of which cefsharp is being used by adapter.

However, when loading the adapter we are getting this error from the adapter 19-09-2023 12:37:33 Status - Error

I have tried to add the bidingredirect in adapter app.config, but that is of no use.

### Expected behavior

since the loading application has downloaded 111 version it should work with that one

### Actual behavior

Could not load file or assembly 'CefSharp.Wpf, Version=105.3.330.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at InSync.eConnect.Http.CustomBrowser..ctor(String firstUrl, Func`2 closingCondition)
   at InSync.eConnect.Http.AuthTypes.Oauth20Client.<GenerateAccessToken>d__4.MoveNext()
VGVjaG5pY2FsIEluZm8gOiBbR2V0TWVzc2FnZSAtIDE4NiAtIExvZ2dlcl0=

### Known Workarounds

Need to add reference to 111 version in adapter.

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by amaitland** [comment]
Converting to discussion as it's not clear there is anything actionable here. A new issue can be created if a clearly actionably item becomes clear.

The project contains a mixed mode `CLI/C++` dll. For the `.Net Framework` projects we rely solely on the default framework loading behaviour. 

> I have tried to add the bidingredirect in adapter app.config, but that is of no use.

Did you add binding redirects to your applications `app.config`? See https://github.com/cefsharp/CefSharp/issues/3750#issuecomment-924465391 for related discussion.
